shared/consts.go

```diff
@@ -16,6 +16,6 @@ const (
 	LogLevelEnvVar                   = "LOG_LEVEL"
 	BasenineHost                     = "127.0.0.1"
 	BaseninePort                     = "9099"
-	BasenineImageRepo                = "ghcr.io/up9inc/basenine"
-	BasenineImageTag                 = "v0.3.0"
+	BasenineImageRepo                = "docker.io/up9inc/basenine"
```

Comment:
any reason to specify "docker.io", I think this can be just "up9inc/basenine"..

Reply:
@gadotroee it's better be `docker.io` though. To clarify it's Docker Hub.

Reply:
Ok, I thought it might be a best practice

Reply:
@gadotroee if a relative path to the default registry has any use case, like a setup that alters the default registry to a mirror. Then we can make it;

```suggestion
	BasenineImageRepo                = "up9inc/basenine"
```

Is there such a use case?

Reply:
I don't think so, probably better to be specific in this case